### ISSUE TITLE
Not aligned with cursor or minimap

### ISSUE THREAD

**Issue by dlewand691** [olmokramer/atom-minimap-codeglance#41]
The codeglance preview snaps to the top or bottom of the screen and doesn't not track to the cursor. Have tried adjusting various settings and reinstalling. No luck with anything so far. Has anyone else experienced this?

OSX 10.13
Atom 1.20.0

**Comment by CallumHoward** [comment]
I'm also getting this
MacOS 10.13
Atom 1.20.1

**Comment by heisian** [comment]
I can confirm this is true. My line height is set to very tall, about double the default line height. Codeglance appears as if it's tracking the code as if it were in its original line height setting.

**Comment by olmokramer** [comment]
Hi everyone, I'm sorry for the extremely late reply. As you have probably noticed I've stopped maintaining my Atom plugins a few years ago. I got very frustrated with how Atom was maintained and started ignoring all of my plugins. I'm sorry for that. I won't install Atom to test and mergeer PRs, but I'd be happy to transfer my repositories to anyone that wants to maintain any of my plugins. Again, I'm sorry.
